.github/workflows/release.yml

```diff
@@ -54,16 +54,10 @@ jobs:
     name: Build binaries
     needs: [release-please]
     if: ${{ needs.release-please.outputs.tag-name }}
-    runs-on: ubuntu-latest
-    steps:
-      - name: Dispatch to build-nargo
-        uses: benc-uk/workflow-dispatch@v1
-        with:
-          workflow: publish.yml
-          repo: noir-lang/build-nargo
-          ref: master
-          token: ${{ secrets.NOIR_REPO_TOKEN }}
-          inputs: '{ "noir-ref": "${{ needs.release-please.outputs.tag-name }}", "publish": true }'
+    uses: ./.github/workflows/publish.yml
+    with:
+      tag: ${{ needs.release-please.outputs.tag-name }}
+      publish: true
```

Comment:
I don't this this actually works because we don't specify the `workflow_call` event. I also believe that it would need to take a secret passed through because I don't think it inherits secrets, but I'm not clear on that.

I'd probably leave this to a workflow_dispatch even if it is just dispatching to the same repo (it should be able to use `GITHUB_TOKEN` now though

Reply:
Ah, `workflow_call != workflow_dispatch`. Good catch.